### PR TITLE
Views

### PR DESCRIPTION
**Pull request by ahmedadel56**
- [x] Implement the design from the sneak peek wireframes.

- Styling is nice to have but focus on making sure everything functions first.

- [x] Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

- [x] Add seeds file for testing

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @ahmedadel56, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The PR title and description looks professional ✔️
- No linter errors 👍
- Descriptive commit messages 👍
- You have a descriptive readme file 👍

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly make use of the`recent_three_posts` method in the user model, let's use that function in our user show view to render the most recent posts.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -2,4 +2,10 @@
 <% @posts.each do |post| %>
   Title: <%= post.title %><br>
   Auther id: <%= post.user_id %><br>
+        <% post.comments.each do |comment| %>
```
[the remark this answers is not in the document]
Comment (ahmedadel56) [reply app/views/posts/index.html.erb:5]: @elisha2kyakpo1  It is already done from the controller itself, please go to post controllers and you will find It is already invoked in the show method

[the remark this answers is not in the document]
Comment (elisha2kyakpo1) [reply app/views/posts/index.html.erb:5]: @ahmedadel56, Great sorry I didn't see that.

[the remark this answers is not in the document]
Comment (ahmedadel56) [reply app/views/posts/index.html.erb:5]: @elisha2kyakpo1  would you give me the acceptance or do I have to request a code review again?

[the remark this answers is not in the document]
Comment (elisha2kyakpo1) [reply app/views/posts/index.html.erb:5]: @ahmedadel56, Sorry you have to make a request again.

**Review by rotimiazeez — APPROVED** [review]
# Hi @ahmedadel56 👍🏾, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**